cmd/tailscale/cli/file.go

```diff
@@ -140,7 +141,17 @@ func runCp(ctx context.Context, args []string) error {
 				return err
 			}
 			if fi.IsDir() {
-				return errors.New("directories not supported")
+				name = fi.Name() + ".zip"
```

Comment:
We wouldn't want to leave the zip file in their filesystem, and ideally it would work by streaming the data from the ZipWriter without writing out a file at all.

Reply:
Sure I'm looking into it.